redundant-bmc/src/manager.cpp

```diff
@@ -53,8 +51,11 @@ Manager::Manager(sdbusplus::async::context& ctx,
 // NOLINTNEXTLINE
 sdbusplus::async::task<> Manager::startup()
 {
-    co_await sdbusplus::async::execution::when_all(services->init(),
-                                                   sibling->init());
+    auto& services = providers->getServices();
```

Comment:
Can we not have an aggregation reference member for these providers?

no harm, just checking...Looks like we are creating a local reference in more places.

Reply:
I think for now I'd like to just pass around the Providers class, so that as new ones are added I don't have to change constructors.  I can revisit when they are all in.